core/src/main/java/ar/com/wolox/wolmo/core/util/NavigationUtils.kt

```diff
@@ -0,0 +1,224 @@
+package ar.com.wolox.wolmo.core.util
+
+import android.app.Activity
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import android.text.TextUtils
+import android.view.View
+import androidx.core.app.ActivityCompat
+import androidx.core.app.ActivityOptionsCompat
+import androidx.core.util.Pair
+import java.io.Serializable
+import java.util.ArrayList
+
+class NavigationUtils {
+
+    companion object {
+        private const val BLANK_PAGE = "about:blank"
+
+
+        /**
+         * Opens the browser with a given URL.
+         *
+         * @param context An instance of [Context]. Can't be null.
+         * @param url     The URL that the browser should open. If the URL is null or empty,
+         * the browser will be opened with a blank page.
+         */
+        fun openBrowser(context: Context, url: String?) {
+            var urlValue = url
```

Comment:
Same as before, don't use `var` unless it's necessary. We could use the if return value.

Reply:
ok :+1: